epictrack-web/src/components/shared/specialField/index.tsx

```diff
@@ -312,6 +313,7 @@ export const SpecialFieldGrid = ({
         background: Palette.white,
         overflowX: "auto",
         position: "absolute",
+        zIndex: theme.zIndex.modal + 1,

```

Comment:
why are adding this?

Reply:
its being blocked by other components on test env